media/icon/dark/refresh.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please insert origin address here too T.T

Reply:
I didn't update yet. :) I'm waiting for your response about https://github.com/Samsung/ONE-vscode/pull/446#discussion_r845682311.
I will update ASAP. :)